.github/workflows/release.yml

```diff
@@ -0,0 +1,45 @@
+name: release
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - 'master'
+    paths:
+      - 'usr/local/share/bastille/version.sh'
```

Comment:
Seems not the perfect place to live, since Bastille expects actual commands in that path. Maybe a .version file in the root or in `util/version.sh` if that file will contain any logic.

Reply:
I see: like if someone issues `bastille version` and it would do unexpected things. But I don't think the reasoning holds because:
1. `common.sh` also lives in `/usr/local/share/bastille` and is not a command
2. the entry point script takes extra care about [what commands are allowed](https://github.com/foudfou/bastille/blob/f8ef0d35bf7fa6c9fe7dffc374a430b5b0985921/usr/local/bin/bastille#L142) and explicitly handles the case of `version`.

I guess the alternative would be to move `common.sh` and `version.sh` elsewhere. But I'm unsure where actually. For the sake of simplicity I would thus keep them in the main bastille lib dir.

Reply:
I understand your argument about `common.sh` but take in mind that today `bastille version` actually does something, just that the `version.sh` file does not exist since the subcommand is directly handled in the `/usr/local/bin/bastille` file.

That said, in order to touch as little files as possible and not create a confusion for later contributions, I'll still suggest to hold that version info file in another location or file.

The `include` directory is not a bad idea for the `common.sh` file and later for the `version.sh` file if you want, but that will translate in a refactoring, so maybe we can check that later since that will need a lot of testing to ensure nothing broke.